modules/nw-cudn-cr.adoc

```diff
@@ -47,8 +47,8 @@ metadata:
 spec:
   namespaceSelector: # <2>
     matchLabels: # <3>
-    - "<example_namespace_one>":"" # <4>
-    - "<example_namespace_two>":"" # <4>
+      "<example_namespace_one>":"" # <4>
```

Comment:
<pre>
"label key1": "label value1"
"label key2": "label value2"
</pre>
or 
<pre>
"key1": "value 1"
"key2": "value 2"
</pre>

Reply:
It should be now fixed